### PR TITLE
bug fixes for bar graphs

### LINKED ISSUE #58: Hill-HarrisX (2019-09-21) not displaying in bar chart
Not yet sure if this is an issue with the code or with the spreadsheet, but likely the spreadsheet
image

### PR DESCRIPTION
**Pull request by fedderw**
fixes #58

### CODE REVIEW

File: py/visualize.py [hunk 1]
```diff
@@ -288,7 +295,8 @@ def wrap_labels(labels, n):
                     x=xd[0] / 2,
                     y=1.15,
                     text=top_labels[0],
-                    font=dict(family="Arial", size=14, color="rgb(67, 67, 67)"),
+                    font=dict(family="Arial", size=14,
```
Comment (MaxGhenis) [inline py/visualize.py:298]: Could we use Roboto throughout?

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
needs `black`

File: py/visualize.py (hunk 1 above)
Comment (fedderw) [reply py/visualize.py]: Here's Arial:
image

and here's Mr. Roboto:
image

Arial looks better on the charts IMO

**Comment by fedderw** [comment]
I moved the font question to an issue, going to merge this

**Pull request merged**